go.mod

```diff
@@ -70,6 +70,7 @@ require (
 	go.etcd.io/etcd v0.5.0-alpha.5.0.20200824191128-ae9734ed278b
 	go.uber.org/atomic v1.7.0
 	go.uber.org/automaxprocs v1.2.0
+	go.uber.org/goleak v0.10.0
```

Comment:
remove this

Reply:
Thanks for your review, I agree with you.

Reply:
This haven't been resolved.